### ISSUE TITLE
Criteria Type Code Codelist, used in v1.0.2 and also in V2.0.x does not exist as an artifcact

### ISSUE THREAD

**Issue by jerouris** [OP-TED/ESPD-EDM#136]
It seems that the Criteria Type Code codelist is not available any more in 2.0.x, although there is a 2.0.2 mention of the list. 
eg. here is a snippet from an example present in the dist folder:
...
``
<cac:TenderingCriterion>
		<cbc:ID schemeID="CriteriaTaxonomy" schemeAgencyID="EU-COM-GROW" schemeVersionID="2.0.1">005eb9ed-1347-4ca3-bb29-9bc0db64e1ab</cbc:ID>
<cbc:CriterionTypeCode listID="CriteriaTypeCode" listAgencyID="EU-COM-GROW" listVersionID="2.0.1">CRITERION.EXCLUSION.CONVICTIONS.PARTICIPATION_IN_CRIMINAL_ORGANISATION</cbc:CriterionTypeCode>
``
...

Where should we get the values of this codelist for version 2.0.x?

**Comment by paulakeen** [comment]
You're absolutely right! this code list was removed from the spread-sheets-book of Code Lists because we it's also in the CriteriaTaxonomy spread-sheets and we did not want to duplicate the maintenance. 

I propose to retrieve it back into the CodeLists spread-sheet, even if that means not forgetting to update both lists when adding/modifiying the codes. One good reason, at least, for this decision is that the automated generation of GeneriCode files is done based on this spread-sheet. 

We'll do it in v.2.0.3.
